doc/architecture/decisions/0005-data-retention-policy.md

```diff
@@ -0,0 +1,100 @@
+
+## Decision
+Here are diagrams illustrating our current and proposed data retention policies - 
+
+> Please note that any events hitting the terminal line indicate they are held indefinately
+
+```mermaid
+gantt
+    title Current Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 35d
+    Session backups: after uj1, 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 2022-12-31
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 2022-12-31
+    Reconciliation Event Metadata: 2022-01-14, 2022-12-31
+    
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+```mermaid
+gantt
+    title Proposed Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 210d
+    Lowcal Session (Sanitised): 2022-12-31
+    Session backups: after uj1, 182d
+    Session backups (Sanitised): 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 182d
+    Submission audit logs (Sanitised): 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 182d
+    Reconciliation audit logs (Sanitised): 2022-12-31
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 182d
+    Reconciliation Event Metadata: 2022-01-14, 182d
+
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+Here are the broad catergory of changes - 
+
+|| Current Retention Period | Proposed Retention Period |
+|---|---|---|
+| **User Data**| Lowcal Sessions - 7 days, then deleted. Session Backups - Indefinite | 6 months, then sanitised and held indefinitely |
+| **Audit Logs** | Indefinite | 6 months, then sanitised and held indefinitely |
+| **Event Metadata** | Indefinite | 6 months, then deleted | 
+| **Feedback** | Indefinite | Indefinite
+
+### What does "Sanitation" mean?
+Currently, we would simply remove / delete and user data (e.g. email, breadcrumbs, passport) whilst retaining the remainder of the metadata associated with a record (such as `id`, `created_at`). This would allow to still have enough information to resconstruct user journeys if required to do so for auditing purposes.
+
+In future, a more fine grained approach could be implemented to only sanitise personally identifiable data using tags.
+
+## Questions
+ - Is 6 months too long?
+ - Do we actually need Session Backups now that we have Lowcal Sessions?
+ - Is there value in retaining reconciliation events indefinately?
```

Comment:
One more question I'd add to this list: "When does this retention policy take effect?" Eg once merged are we immediately sanitising records `created_at` more than 6 months ago? Or does the first sanitisation happen 6 months from merge? 

The first options seems ideal and sensible to me, but feels worth asking & being explicit about since this has been a fuzzy topic for so long!